frame/dapps-staking/src/weights.rs

```diff
@@ -29,7 +31,7 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: DappsStaking RegisteredDapps (r:1 w:1)
 	// Storage: DappsStaking PreApprovalIsEnabled (r:1 w:0)
 	fn register() -> Weight {
-		(32_139_000 as Weight)
+		(60_000_000 as Weight)
```

Comment:
We'll have to see about all these changes...we have a problem with current weights.

I ran them on my laptop which is more powerful than the referent machine which should be used to measure weights. So currently, all of our weights are a bit too small!

But it seems your execution increased weights quite a bit - so maybe your machine is closer to the referent machine specs. Did you ensure nothing else is running when you ran benchmarks?

Anyhow, you could try benchmarking `pallet-balances` to generate your personal weights file and then compare it to the official weights in substrate repo. That should tell you in a sense how close your machine to referent machine is.

Reply:
I tried benchmarking `pallet_balances`, and I also get the double that of the official repo.
I'm running it on macbook pro, 2,3 GHz 8-Core Intel Core i9, 16 GB 2667 MHz DDR4, which is considerably weaker than the reference machine.

Reply:
Oh!

In that case, perhaps it's better I run benchmarks on my machine just so we keep it consistent.